app/src/main/res/layout/activity_splash.xml

```diff
@@ -35,6 +35,7 @@
         android:background="@android:color/transparent"
         android:drawableStart="@drawable/create_contest"
         android:text="@string/create_contest"
+        android:textColor="@color/contestAppTextColor"
```

Comment:
I think Abass will take care of this in his look-and-feel branch; I only added this color because I changed the default button textColor to white (but the splash page buttons are different than the default ones).